pkg/genericapiserver/genericapiserver.go

```diff
@@ -128,6 +131,20 @@ type APIGroupVersionOverride struct {
 	ResourceOverrides map[string]bool
 }
 
+// Info about an API group.
+type APIGroupInfo struct {
+	GroupMeta latest.GroupMeta
+	// Info about the resources in this group. Its a map from version to resource to the storage.
+	VersionedResourcesStorageMap map[string]map[string]rest.Storage
```

Comment:
map[string] implies there is no ordering in the versions, but there is.  This should really be []structThatIncludesMapStringToStorage